### ISSUE TITLE
Next release?

### ISSUE THREAD

**Issue by pestaa** [nextcloud/ocsms#319]
Hi!

Is there a release date scheduled?

I'm on the latest stable and affected by #297. The fix is 10 days old so I can wait. :)

Thank you for this great project. You just made online payment SMS verification so much better.

**Comment by e-alfred** [comment]
@nerzhul is the only one who can push new releases to the App Store which is necessary anyway for Nextcloud 19.

**Comment by e-alfred** [comment]
@nerzhul I tested the app with the latest RC of Nextcloud 19, can you make a new release for Nextcloud 19 to the app store?

**Comment by nerzhul** [comment]
didn't i gave you the key or transfered the app permissions on the store ?

**Comment by e-alfred** [comment]
I thought it would be only for the Weather app. I will do a release if the key works for both apps.

**Comment by shinenelson** [comment]
I see that there's a [new release on the app store](https://apps.nextcloud.com/apps/ocsms/releases?platform=19#19) but it lists version v1.10.1 which is not even listed even for Nextcloud 9. I was wondering if that was an official release with the version number messed up or is there something else that is messed up here?

**Comment by e-alfred** [comment]
I published release 2.1.8, @nerzhul please publish it to the App Store.

**Comment by shinenelson** [comment]
Wait, so, what's already on the app store for Nextcloud version 19?

**Comment by nerzhul** [comment]
release published, sorry for the delay

**Comment by shinenelson** [comment]
can someone clarify why there are 2 major versions being released for a single Nextcloud release version?

![image](https://user-images.githubusercontent.com/4771718/85975747-7ee12880-b9f6-11ea-99b4-49a59a1f3c41.png)

I just noticed that this one version - 1.10.1 has been there in all the prior releases.

I get when there are new development and minor versions are released but what's this version that's way old? And it reached the apps store even before this new version ( 2.1.8 ) was released. I doubt it'll even work in the latest versions.
